models/code_climate.py

```diff
@@ -13,37 +35,65 @@ def getDesc(self) -> Dict:
                 "complex_logic": "./metrics/complex_logic.py",
                 "method_count": "./metrics/method_count.py",
                 "return_statements": "./metrics/return_statements.py",
-                "identical_code": "./metrics/identical_codeblocks.py",
-                "similar_code": "./metrics/similar_codeblocks.py",
+                "identical-code": "./metrics/identical_codeblocks.py",
+                "similar-code": "./metrics/similar_codeblocks.py",
```

Comment:
why are these with ´-´ when everything else is snake casing?

Reply:
It matches with the output of Code Climate's API, makes comparison just a little bit easier. If we think this is unnecessary (hidden) coupling I am open to change it.